cognite/seismic/protos/ingest_service_messages.proto

```diff
@@ -117,6 +117,22 @@ message DeleteFileRequest {
 
 // -----
 
+/**
+    [Example]
+    {"survey": {"name": "survey_name"} }
+**/
+message DeleteSurveyRequest {
+    cognite.seismic.Identifier survey = 1; // [required] Either name or id of a survey previously registered with /RegisterSurvey
+    /*
+     If set to true, will only delete metadata (undo the ingestion), but keep the registered files of survey as is.
```

Comment:
Uhm.... delete metadata/undo ingestion... what?

Reply:
It's just consitent with our other delete endpoints that allow to do this. It's weird, and shouldn't be there in V1